tools/code_coverage/coverage_diff.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+
+def get_diff_file_lines(diff_file):
+    diff_file_lines = {}
+    current_file = None
+    current_line = -1
+
+    with open(diff_file) as f:
+        for line in f:
+            line = line.strip()
+            if line.startswith("+++ "):
+                current_file = line[4:]
+                diff_file_lines[current_file] = []
+                continue
+            elif line.startswith("@@ "):
+                current_line = int(line.split()[2].lstrip("+").split(",")[0])
+                continue
+            elif line.startswith("-"):
+                continue
+            elif line.startswith("+"):
+                diff_file_lines[current_file].append(current_line)
+            current_line += 1
+
+    return diff_file_lines
+
+
+def get_info_file_lines(info_file, diff_file):
+    diff_file_lines = get_diff_file_lines(diff_file)
+    current_lines = []
+    current_lf = current_lh = 0
+    base_path = "/work/FlagGems/"
```

Comment:
Again, we need a common.sh to save these paths

Reply:
dito